cirq/ops/identity.py

```diff
@@ -64,35 +64,6 @@ def _qid_shape_(self) -> Tuple[int, ...]:
     def num_qubits(self) -> int:
         return len(self._qid_shape)
 
-    def on_each(self, *targets: Union['cirq.Qid', Iterable[Any]]) -> List['cirq.Operation']:
-        """Returns a list of operations that applies the single qubit identity
-        to each of the targets.
-
-        Args:
-            *targets: The qubits to apply this gate to.
-
-        Returns:
-            Operations applying this gate to the target qubits.
-
-        Raises:
-            ValueError if targets are not instances of Qid or List[Qid] or
-            the gate from which this is applied is not a single qubit identity
-            gate.
-        """
-        if len(self._qid_shape) != 1:
-            raise ValueError('IdentityGate only supports on_each when it is a one qubit gate.')
```

Comment:
heyo!

Reply:
Elaborate! Is that a positive exclamation that "heyo, nice find, look, this works here as well and the code is simpler" or "heyo don't do this"? :)

Reply:
> heyo, nice find, look, this works here as well and the code is simpler

This one!

Reply:
Ack. Thanks!